packages/core/admin/admin/src/content-manager/routes.tsx

```diff
@@ -1,5 +1,15 @@
 /* eslint-disable check-file/filename-naming-convention */
-import { Navigate, RouteObject } from 'react-router-dom';
+import { Navigate, RouteObject, useParams } from 'react-router-dom';
+
+const RedirectToCollectionTypes = ({
```

Comment:
Previous the <Navigate> components were literally going to "/collection-types/:slug". This component lets us get the slug for the redirect. Is there a better way?

Reply:
I'm not sure if there is a better way, did you try with a splat route instead of the `:slug`? 🤔